### ISSUE TITLE
divide vimeo90K dataset into training set and test set

### ISSUE THREAD

**Issue by yaolugithub** [anchen1011/toflow#16]
Thank you very much for your work.  It is a extremely good work .
I have a problem about how to divide vimeo90K dataset into training set and LR images?

I have seen that there is a get_path_sep.m in src/evalution and sep_trainlist.txt in data. 
I want to know how to call to generate the corresponding training set ?

Thanks a lot.
